dev-frontend/src/pageComps/homePage.js

```diff
@@ -80,28 +82,95 @@ class HomePage extends React.Component{
         return(
             <div
                 id='appContents'
-                className='color-background_primary color-text_primary margin_horizontal4'
+                className='color-background_primary color-text_primary'
                 style={{overflowY:'hidden'}}>
-                <div>
-                  <label>City Filter</label>
-                  <input
-                      onChange = {this.handleCityFilter}
+                <div
+                  id='headerLayoutDiv'
+                  class='marginHeader'>
+                  <HeaderComp
+                    headerStyle={2}
                   />
-                  <label>Sport Filter</label>
-                  <input
-                      onChange = {this.handleSportFilter}
+                </div>
+                <div
+                  id='bodyLayoutDiv'
+                  class='margin_horizontal4'>
+                  <div
+                    id='logoDiv'
+                    class='flex-column flex_center homeLogoDiv'>
+                    <img
+                      src={Logo}
+                      class='homeLogo'>
+                    </img>
+                  </div>
+                  <div
+                    id='searchDiv'
+                    className='flex-column flex_center'>
+                    <div 
+                      className='searchDiv flex-column'>
+                      <input 
+                        id='searchBar'
+                        type="text"
+                        placeholder="Search for a team.."
+                        maxLength="40"
+                        className='searchBar'/>
+                      <div
+                        id='filterBar'
+                        className='flex-row filterBar'>
+                        <div
+                        className='filterDiv'
+                          style={{'flex':1}}>
+                          <span>Filter</span>
+                        </div>
+                        <select
```

Comment:
All of these options will need to come from the backend eventually right?

Reply:
Yeah all of the selects will probably come from the back-end.